subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenPomPackagingResolveIntegrationTest.groovy

```diff
@@ -333,4 +333,27 @@ compile('group:projectA:1.0') {
         succeeds 'retrieve'
     }
 
+    @Issue('https://github.com/gradle/gradle/issues/1789')
+    def "ignore artifact when relocation is defined in pom"() {
+        given:
+        buildFile << """
+repositories { jcenter() }
+configurations { compile }
+dependencies {
+    // http://repo1.maven.org/maven2/org/apache/commons/commons-io/1.3.2/commons-io-1.3.2.pom
+    compile 'org.apache.commons:commons-io:1.3.2' 
```

Comment:
Instead of reaching out to external dependencies we should recreate the scenario with `MavenHttpModule`. The reason is that external calls can easily fail a CI build. We'd have to introduce some new functionality in case the `MavenHttpModule` doesn't support a method for relocation yet.